app/react/ToggledFeatures/tocGeneration/FilterTocGeneration.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Aggregations } from 'shared/types/Aggregations.d.ts';
+import { FeatureToggle } from 'app/components/Elements/FeatureToggle';
+import SelectFilter from 'app/Library/components/SelectFilter';
+import FormGroup from 'app/DocumentForm/components/FormGroup';
+import { t } from 'app/I18N';
+import { NeedAuthorization } from 'app/Auth';
+
+export interface FilterTocGenerationProps {
+  onChange: () => void;
+  aggregations: Aggregations;
+}
+
+const filteredAggregation = (aggregations: Aggregations, key: string) => {
+  const bucket = (aggregations?.all?.generatedToc?.buckets || []).find(a => a.key === key) || {
+    filtered: { doc_count: 0 },
+  };
+  return bucket.filtered.doc_count;
+};
+
+const options = (aggregations: Aggregations = { all: {} }) => [
+  {
+    label: 'Automatically generated',
```

Comment:
should we use t function to make those labels translatable at least in the translations form? or maybe in another issue update the Multiselect component to wrap labels arround <Translate> ?

Reply:
done, i have translated this labels